### ISSUE TITLE
performance without turnbert

### ISSUE THREAD

**Issue by Chenfeng1271** [zyang-ur/ReSC#3]
Dear Sir:
 I rerun the code without tunebert using bert-base on referit and unc, but the final results are 66.50/62.53 and 75.30/- for val/test on these two datasets. Should I use tunebert in all training to achieve the performance as reported?

**Comment by zyang-ur** [comment]
Solved offline. Due to hyperparameter setting.

**Comment by MI-WENYU** [comment]
> Solved offline. Due to hyperparameter setting.

Hello, I set Tunebert =False in the training process, but the accuracy can only reach 20% in the referit data set.  The accuracy can reach tunebert=true in the paper. Does this happen when you do your experiments?

**Comment by MI-WENYU** [comment]
> 



> Dear Sir: I rerun the code without tunebert using bert-base on referit and unc, but the final results are 66.50/62.53 and 75.30/- for val/test on these two datasets. Should I use tunebert in all training to achieve the performance as reported?

Hello, how many epochs did you run to get this result?
